.travis.yml

```diff
@@ -6,6 +6,10 @@ cache:
   directories:
     - $HOME/.composer/cache
 
+env:
+  - STABILITY=''
+  - STABILITY='--prefer-lowest'
```

Comment:
it would be much better to run a single job for prefer-lowest instead of one per PHP version, to limit the size of the build matrix. Travis allows only 5 concurrent jobs per repo owner (i.e. for all repos of @schmittjoh in this case) so the size of the matrix has an impact